make/os_mac

```diff
@@ -14,17 +14,15 @@ ifeq (g++,$(CC_TYPE))
 endif
 
 ifeq (clang++,$(CC_TYPE))
-  CFLAGS_GTEST += -Wc++11-extensions
-  CFLAGS_GTEST += -Wno-c++11-long-long
   CFLAGS += -Wno-unknown-warning-option
   CFLAGS += -Wno-unused-function
   CFLAGS += -Wno-tautological-compare
-  CFLAGS += -Wno-c++11-long-long
   CFLAGS += -Wsign-compare
   CFLAGS += -Wno-unused-local-typedef
-  ifeq (true,$(C++11))
-    CFLAGS += -stdlib=libc++ -std=c++11
-  endif
+
+  #https://stackoverflow.com/questions/19774778/when-is-it-necessary-to-use-use-the-flag-stdlib-libstdc
+  #for OSX before mavericks, specifically, but fine for post-mavericks too
+  #CXXFLAGS += -stdlib=libc++
   #PCH = .pch
   ifeq (4.2,$(CC_MAJOR).$(CC_MINOR))
```

Comment:
I think we should just remove it. I just took a look at that file and there's a repeated block in the g++ section that could be removed too.